abilities/hre/relic-garrisoned-keep-3.json

```diff
@@ -0,0 +1,33 @@
+{
+  "id": "relic-garrisoned-keep-3",
+  "baseId": "relic-garrisoned-keep",
+  "type": "ability",
+  "name": "Religious Zeal Keeps and Outposts",
+  "pbgid": 195371,
+  "attribName": "relic_tower_keep_bonus_hre",
+  "age": 3,
+  "civs": [
+    "hr"
+  ],
+  "description": "• Armor increased by +50%\n• Damage increased by +35%\n• Sight range increased by +25%\n• Weapon range increased by +20%",
+  "classes": [],
+  "displayClasses": "",
+  "costs": {
+    "food": 0,
+    "wood": 0,
+    "stone": 0,
+    "gold": 0,
+    "total": 0,
+    "popcap": 0
+  },
+  "producedBy": [
+    "outpost",
+    "stone-wall-tower",
+    "keep",
+    "elzbach-palace"
+  ],
+  "icon": "https://data.aoe4world.com/images/abilities/relic-garrisoned-1.png",
+  "active": "always",
+  "auraRange": 0,
+  "effects": []
```

Comment:
Effects missing

Reply:
Honestly, I have not done any effects of this next round of abilities discovered by this latest commit.  Will work on that soon.

Reply:
@robertvanhoesel this one has an issue adopting the modifiers because relic_dock_bonus_hre and relic_tower_keep_bonus_hre have the same baseId when modifiers are added; before I change the ID in a workaround.  

I think attribName is safer to call in modifier.  I can change the 50 abilities to reference attribName instead.

Reply:
Hmm, using the attrib name isn't a bad idea but may make it a bit harder to navigate the files. Since some of the attrib names are chosen pretty poorly. I'd recommend also including the title or base ID in the comments when switching to attrib name